<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetLightCommunicationDlg</class>
 <widget class="QWidget" name="NetLightCommunicationDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="LightCom_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>控制器IP设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_19">
      <item row="0" column="0">
       <widget class="QWidget" name="checkBox_light_crl_1" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_20" columnstretch="0,0,0,0,0,0,0">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item row="4" column="1">
          <widget class="QLineEdit" name="lineEdit_ip_crl_2">
           <property name="inputMask">
            <string>000.000.000.000</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>IP地址</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QSpinBox" name="spinBox_port_crl_0">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_55">
           <property name="text">
            <string>控制器编号</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_ip_crl_0">
           <property name="inputMask">
            <string>000.000.000.000</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_58">
           <property name="text">
            <string>端口</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>  编号3</string>
           </property>
          </widget>
         </item>
         <item row="4" column="4">
          <widget class="QCheckBox" name="checkBox_light_crl_3">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QSpinBox" name="spinBox_port_crl_2">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
         <item row="4" column="5">
          <widget class="QPushButton" name="pushButton_getCtrType_crl_2">
           <property name="text">
            <string>测试通讯</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QCheckBox" name="checkBox_light_crl_0">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="5">
          <widget class="QPushButton" name="pushButton_getCtrType_crl_0">
           <property name="text">
            <string>测试通讯</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>使能</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>  编号2</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>  编号1</string>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QCheckBox" name="checkBox_light_crl_2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QSpinBox" name="spinBox_port_crl_1">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
         <item row="3" column="5">
          <widget class="QPushButton" name="pushButton_getCtrType_crl_1">
           <property name="text">
            <string>测试通讯</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_ip_crl_1">
           <property name="inputMask">
            <string>000.000.000.000</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>  编号4</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="lineEdit_ip_crl_3">
           <property name="inputMask">
            <string>000.000.000.000</string>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QSpinBox" name="spinBox_port_crl_3">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
         <item row="5" column="4">
          <widget class="QCheckBox" name="checkBox_light_crl_4">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="5" column="5">
          <widget class="QPushButton" name="pushButton_getCtrType_crl_3">
           <property name="text">
            <string>测试通讯</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <spacer name="horizontalSpacer_24">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>18</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushBtnIp_ok">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>保存</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushBtnIp_cancel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>修改</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_10">
     <property name="title">
      <string>通讯测试</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>控制器编号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_ctrl_index">
          <item>
           <property name="text">
            <string>编号1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>编号2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>编号3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>编号4</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_63">
            <property name="text">
             <string>控制器(net_id)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_net_id">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>2</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_64">
            <property name="text">
             <string>命令</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_order">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="value">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_72">
            <property name="text">
             <string>静态触发频率</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_frequence">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
            <property name="value">
             <number>1000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="label_69">
            <property name="text">
             <string>总脉冲数1</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinBox_all_pulse_1">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
            <property name="value">
             <number>12800</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_65">
            <property name="text">
             <string>总脉冲数2</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QSpinBox" name="spinBox_all_pulse_2">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_66">
            <property name="text">
             <string>触发间距1</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBox_pulse_number_1">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_67">
            <property name="text">
             <string>触发间距2</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QSpinBox" name="spinBox_pulse_number_2">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_68">
            <property name="text">
             <string>延迟脉冲1</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="spinBox_delay_pulse_1">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_70">
            <property name="text">
             <string>延迟脉冲2</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QSpinBox" name="spinBox_delay_pulse_2">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_send_st">
          <property name="text">
           <string>发送</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="2,1">
        <item>
         <layout class="QGridLayout" name="gridLayout_6" rowstretch="0,0">
          <item row="0" column="0">
           <widget class="QLabel" name="label_71">
            <property name="text">
             <string>控制器</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_73">
            <property name="text">
             <string>命令</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_74">
            <property name="text">
             <string>操作 </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSpinBox" name="spinBox_t_net_id">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="value">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBox_t_order">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="spinBox_t_crl">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_send_order">
          <property name="text">
           <string>发送</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
